front/src/applications/operationalStudies/v2/components/ScenarioHeader.tsx

```diff
@@ -0,0 +1,51 @@
+import { X, ChevronDown, Question } from '@osrd-project/ui-icons';
+
+import useAuth from 'utils/hooks/useAuth';
+
+type Props = {
+  scenarioName: string;
+};
+
+const ScenarioHeader = ({ scenarioName }: Props) => {
+  const { username } = useAuth();
+
+  const boards = ['Traits', 'Map', 'Macro', 'STD', 'SDD', 'Table', 'Conflicts'];
+
+  return (
+    <header className="scenario-header">
+      {/* A — scenario info */}
```

Comment:
Are these letters referring to something ?
If not we could just use that (nit)
```suggestion
      {/* scenario info */}
```